pkg/iam/oidc/api.go

```diff
@@ -16,10 +16,7 @@ import (
 	cft "github.com/weaveworks/eksctl/pkg/cfn/template"
 )
 
-var defaultAudienceByPartition = map[string]string{
-	"aws":    "sts.amazonaws.com",
-	"aws-cn": "sts.amazonaws.com.cn",
-}
+var defaultAudience = "sts.amazonaws.com"
```

Comment:
Please make this a const
```suggestion
const defaultAudience = "sts.amazonaws.com"
```